webots_ros2_epuck/webots_ros2_epuck/driver.py

```diff
@@ -281,6 +281,7 @@ def step_callback(self):
         self.publish_distance_data(stamp)
```

Comment:
This Is not directly related to this PR, but I just saw a small issue at line 101, this is duplicated:
```
self.camera_period = camera_period_param.value
```